helper/helper.py

```diff
@@ -7,7 +7,7 @@
 DB_PASSWORD = 'root' # noqa
 
 
-def get_db_session(db_host = 'localhost:5432', db_user = 'root'):
+def get_db_session(db_host='db:5432', db_user='root'):
```

Comment:
Since developers mostly run database with docker using `db` is a more sensible default.

Reply:
Agreed. Perhaps let's just mandate docker now for consistency, and add config options later if an actual use case arises where someone would need to run it all raw.